packages/dds/cell/src/cell.ts

```diff
@@ -278,7 +281,9 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>>
         return this.serializer.decode(value);
     }
 
-    protected applyStashedOp() {
-        throw new Error("not implemented");
+    protected applyStashedOp(content: any) {
+        this.applyInnerOp(content);
+        ++this.messageId;
+        return { type: "edit", pendingMessageId: this.messageId };
```

Comment:
Take a look at what SharedCell expects the type of the localOpMetadata to be in processCore(). The metadata returned to runtime from applyStashedOp() here will be passed back to processCore() once the op has been sequenced by the server.